cogs/monitor.py

```diff
@@ -183,34 +213,42 @@ async def message_delete(self, ctx: commands.Context, pattern_id: int):
         get_all_monitor_messages.cache_clear()
 
         logger.debug(f"Removed pattern {monitored_message.message} from monitor.")
-        await ctx.send(f"The pattern (`{monitored_message.message}`) has been successfully removed from monitor.")
+        await ctx.send(
+            f"The pattern (`{monitored_message.message}`) has been successfully removed from monitor."
+        )
 
-    @message.command(name='toggle')
+    @message.command(name="toggle")
     async def message_toggle(self, ctx: commands.Context, pattern_id: int):
         """
         Toggle a monitor for a message pattern.
         """
-        monitored_message = await StaffMonitorMessage.filter(monitor_message_id=pattern_id).get_or_none()
+        monitored_message = await StaffMonitorMessage.filter(
+            monitor_message_id=pattern_id
+        ).get_or_none()
 
         if not monitored_message:
             await ctx.send(f"Pattern with ID `{pattern_id}` does not exist.")
             return
-        
+
         monitored_message.disabled = not monitored_message.disabled
         await monitored_message.save()
         get_all_monitor_messages.cache_clear()
 
-        logger.debug(f"{'Disabled' if monitored_message.disabled else 'Enabled'} pattern with ID {monitored_message.monitor_message_id} in monitor.")
-        await ctx.send(f"The pattern (`{monitored_message.message}`) [ID: {monitored_message.monitor_message_id}] has been successfully {'disabled' if monitored_message.disabled else 'enabled'}.")
+        logger.debug(
+            f"{'Disabled' if monitored_message.disabled else 'Enabled'} pattern with ID {monitored_message.monitor_message_id} in monitor."
+        )
+        await ctx.send(
+            f"The pattern (`{monitored_message.message}`) [ID: {monitored_message.monitor_message_id}] has been successfully {'disabled' if monitored_message.disabled else 'enabled'}."
```

Comment:
Line 241 + line 305 are over the 95 character line length (may involve other lines as well). Should probably manually adjust these.